src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -496,15 +497,26 @@ it back */
 	height: 900px;
 }
 
+:deep(){
+	.modal-wrapper--normal{
+		.modal-container {
+			height: max-content;
+		}
+	}
+}
+
 .navigation {
+	position: sticky;
+    bottom: 0;
 	display: flex;
 	justify-content: space-between;
 	flex: 0 0 40px;
-	height: 50px;
 	background-color: var(--color-main-background);
-	box-shadow: 0 -10px 5px var(--color-main-background);
+	box-shadow:0 -2px 5px var(--color-background-darker);
 	z-index: 1;
-	width: 100%;
+	width: 600px;
+	padding: 10px;
+	height: auto;
```

Comment:
```suggestion
	padding: 20px;
```
![image](https://github.com/nextcloud/spreed/assets/93392545/087140e0-6793-4a1c-87c7-26baf986517f)

Fixed width adds a horizontal scrollbar.
Also height seems to be redundant here.
And I'd like to keep every elements inside the 20px frame.

Reply:
`padding: 10px 20px 20px` or `padding: 10px 20px` ?
Side paddings are more significant there. For the top/bottom, maybe could be reduced. It looked fine for me with small top padding and an old shadow